src/app/modules/content/learning-objects/image/image.component.css

```diff
@@ -1,6 +1,6 @@
 img {
   display: block;
-  max-width: 100%;
+  max-width: 90%;
```

Comment:
The zoom-in functionality of our images does not work. As a side-objective, see if you can find the cause.